test/libsolidity/SyntaxTest.cpp

```diff
@@ -188,6 +189,9 @@ int SyntaxTest::registerTests(
 	}
 	else
 	{
+		static vector<unique_ptr<string>> filenames;
+
+		filenames.emplace_back(new string(_path.string()));
```

Comment:
We should really check whether we cannot just use `make_unique`.

Reply:
https://github.com/ethereum/solidity/blob/7a066efd7e0f371cfaee69c1f6b5f0aed1bfd107/cmake/EthCompilerSettings.cmake#L29

Until the compiler settings are changed to ``-std=c++14`` I think ``make_unique`` is pretty much off the table - the only thing one could do other than bumping the compiler version would be to use a customly defined ``make_unique`` in some shared header...

Reply:
Sure, I mean we should evaluate whether we could change it to `c++14`.

Reply:
Ubuntu trusty has gcc 4.8, which has limited support for c++14 and might need `-std=c++1y`.